docs/editors/vscode.md

```diff
@@ -132,6 +132,65 @@ As you type, the symbol outline is also visible at the top of the file.
 
 ```
 
+## Running and debugging your code
+
+Metals supports running or debugging tests and main methods via
+[Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/),
+used to communicate with the editor, and
+[Bloop](https://scalacenter.github.io/bloop/), which is responsible for starting
+the actual process. This means that applications can be run the same as for any
+other language in the natively supported `Run` view.
+
+Users can begin the debugging session in two ways:
+
+- via code lenses
+
+For each main or test class Metals shows two code lenses `run | debug`, which
```

Comment:
for test class there is `test | debug test` lens

Reply:
Forgot about it, it's added.